ansible/roles/kubernetes-common/tasks/main.yml

```diff
@@ -54,6 +54,12 @@
     src: etc/kubernetes/kubeadm.conf
     dest: /etc/kubernetes/kubeadm.conf
 
+- name: drop api-server audit policy template
+  template:
+    dest: /etc/kubernetes/apiserver-audit-policy.yaml
+    src: etc/kubernetes/apiserver-audit-policy.yaml
+  when: "'masters' in group_names"
```

Comment:
Should this be in the kubernetes-master role instead of having this conditional?

Reply:
Hmmm yeah. Dropped it here because I added the kubeadm audit flags in this role. I agree it's better to move it to the `kubernetes-master` role.